### DISCUSSION TITLE
ChakraUI defaulting to fallback fonts

### DISCUSSION THREAD

**Discussion by X-Equals-Zero** [chakra-ui/chakra-ui#6386] [Q&A]
I am running this inside of a NextJS application. The way I understand it is that for the ChakraUI library, you use {extendTheme} to add your own styling configuration, and when you you define your font-families, you also include a fallback font (as required according to the documentation, ['Please do not forget to add font-family fallbacks like e.g. sans-serif on both options.'](https://chakra-ui.com/community/recipes/using-fonts)) . (e.g. heading: { "first choice", "fallback choice" }) This is what my theme configuration looks like:

``` const theme = extendTheme({
  fonts: {
    heading: '"Droid Serif", sans-serif',
    body: '"Open Sans", sans-serif',
  },
  textStyles: {
    navbar: {
      fontFamily: 'heading',
      textTransform: 'uppercase'
    }
  },
  config,
  styles: {
    global: {
      body: {
        bg: 'gray.100',
      }
    }
  }
}) 
```

Unfortunately it will automatically default to 'sans-serif', even though if I remove that fall-back font from the theme and just leave 'droid serif', the custom fonts work perfectly fine. How I set the font in my components:

```
            <Text
              textAlign={useBreakpointValue({ base: 'center', md: 'right' })}
              fontFamily={'heading'}
              fontSize='2rem'
              color={'gray.800'}>
              Sample Title
            </Text>```


Any pointers would be greatly appreciated.

**Comment by matteo95g** [comment]
I'm having the same issue, any suggestions?

**Comment by alibacova** [comment]
In case someone else is stuck on this, I had the same issue and what worked for me was the following: 
I followed [instructions](https://nextjs.org/docs/app/building-your-application/optimizing/fonts#using-multiple-fonts) for multiple fonts from Next.js docs, specifically creating a CSS variable for font and using those in ChakraUI theme.

1. Set up CSS variables for your fonts where you import them from `next/font` as suggested in Next.js docs
2. In your case, add ``className={`${droid_serif.variable} ${open_sans.variable}`}`` to html tag in layout file
3. Use the CSS variables that were set up in step 1 within ChakraUI theme like so 
`fonts: { 
heading: "var(--font-droid-serif), sans-serif", 
body: "var(--font-open-sans), sans-serif"}`
